src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationInvalidSwitchIndent.java

```diff
@@ -98,9 +98,9 @@ private void method1() { //indent:4 exp:4
         switch (s) { //indent:8 exp:8
 	    case 1: //indent:8 exp:12 warn
 	    	if (true) //indent:12 exp:16 warn
-	    		break; //indent:16 exp:20 warn
+                break; //indent:16 exp:20 warn
 	    	else //indent:12 exp:16 warn
-	    		break; //indent:16 exp:20 warn
+                break; //indent:16 exp:20 warn
```

Comment:
please explain this change, we do not expect Input files to be changed.

Reply:
I don't know why, but earlier it was indented at level 20, and was having wrong indent message of `indent:16` therefore the indentation check was not logging any errors there.
Now it is indented at level 16 and exp is 20 so correct error logs are encountered.

Reply:
OK my bad, I have fixed it in my new commit.